setup.sh

```diff
@@ -131,6 +134,12 @@ function install_macos() {
         echo "-------------------------"
         /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     fi
+    #Gets number after decimal in python version
+    version=$(egrep -o 3.\[0-9\]+ <<< $PYTHON | sed 's/3.//g' ) 
+    
+    if [[ $version -ge 9 ]]; then               #Checks if python version >= 3.9
+        install_mac_newer_python_dependencies
+    fi
     install_talib
     test_and_fix_python_on_mac
```

Comment:
I'm not convinced this method (or rather - the content of this method) is still necessary / makes sense.

first of all - it's using python3.6 (so this method stems from a time where 3.6 was the latest version) - and i THINK it's been fixing python2.7 issues (where mac still had 2.7 as primary python installation).

Reply:
Are you talking about `install_mac_newer_python_dependencies` or ` test_and_fix_python_on_mac`?

Reply:
`test_and_fix_python_on_mac` ... (the old / existing one) 

the new one i trust you with and assume works fine (as said, can't really test on a mac).